resources/assets/less/bem/page-mode.less

```
@@ -76,6 +83,12 @@
       margin-top: 5px;
       vertical-align: middle;
     }
+
+    @media @mobile {
+      &:last-child {
+        padding-right:10px;
```

whoops, missing a space
```suggestion
        padding-right: 10px;
```